usr/share/rear/rescue/default/500_ssh.sh

```diff
@@ -24,10 +24,14 @@ if has_binary sshd; then
     Log "Adding required libfreeblpriv3.so to LIBS"
 
     # copy ssh user
-    if PASSWD_SSH=$(grep ssh /etc/passwd) ; then
+    PASSWD_SSH=$(getent passwd ssh sshd)
```

Comment:
Can it happen that you get two entries back? Will the code still work?

Reply:
As fas as I see the
<pre>
IFS=: read user ex uid gid gecos homedir junk <<<"$PASSWD_SSH"
</pre>
ensures that only the first "getent passwd ssh sshd"
entry gets actually used.

Reply:
@N3WWN 
perhaps an explanatory comment why it works
if "getent passwd ssh sshd" results two entries
could avoid future doubts.

Reply:
@jsmeix 
Explanatory comment added as well as searching for 'sshd' account before 'ssh' account.  

I can't provide any examples where the account is named 'ssh', but that's what ReaR was looking for previously.  Instead of breaking the account search for distros that I'm not familiar with by changing 'ssh' to 'sshd', I search for 'sshd' (which is the username on every system that I've checked) first and then ssh.

This also allows this code to work on systems that have 'sshd' as the system account and a non-system or secondary 'ssh' account.